astrology.lic

```diff
@@ -72,9 +72,10 @@ class Astrology
     result.compact.select { |data| data['circle'] <= DRStats.circle }
   end
 
-  def check_moons
-    %w[xibar yavash katamba].each do |moon|
-      bput("pow #{moon}", 'roundtime')
+  def check_attunement
+    return if DRSkill.getxp('Attunement') > 30
```

Comment:
I already clicked the merge button, but it just occurred to me that you might want to move the Attunement check inside the `.each` loop. What do you think?

Reply:
I thought of that too earlier. but decided against it as it would still kill time off the observation timer.